styles/card.css

```diff
@@ -0,0 +1,153 @@
+.card {
+  background: var(--cardBackground);
+  width: 20rem;
+  border-radius: 0.5rem;
+  cursor: pointer;
+  border: 1px solid transparent;
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.card-box-shadow {
+  box-shadow: 0 4px 4px 0 rgba(1, 1, 0.5, 0.05);
+}
+
+.horizontal-card-img {
+  width: 10rem;
+  height: 8rem;
+  border-top-left-radius: 0.5rem;
+}
+
+.card-img {
+  width: 20rem;
+  height: 12rem;
+  border-top-left-radius: 0.5rem;
+  border-top-right-radius: 0.5rem;
+}
+
+.card-section {
+  display: flex;
+  flex-direction: column;
```

Comment:
You can create a utility class for flex layout, it will provide you more flexibility and also reduce your code duplicacy.